src/draw.coffee

```diff
@@ -202,20 +207,24 @@ exports.Draw = class Draw
         }
 
       _clearCache: ->
+        @_cacheFlag = true
         if @_cacheFlag
-          minX = minY = Infinity
-          maxX = maxY = 0
-          for point in @_points
-            minX = min minX, point.x
-            maxX = max maxX, point.x
+          if @_points.length is 0
```

Comment:
Didn't read this too carefully yet - but a general question - why does any caching logic have to change due to the floating assembly change?